meteor-app/imports/ui/layouts/main/component.js

```diff
@@ -11,19 +11,22 @@ export default class MainLayout extends React.Component {
     const {
       store,
       body,
+      helpUrl,
     } = this.props;
+
     return (
       <div className="page layout-main">
         <div className="page__header">
-          <p>Some header</p>
```

Comment:
So that you can see and adjust the new layout of the header and footer without having unnecessary things there.